api/userRoutes/users.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express");
+const router  = express.Router();
+const User  = require("../models/models")
+const Parse = require('parse/node')
+
+// Get list of favorited cities
+router.get("/", async (req, res, next) => {
```

Comment:
If this route is supposed to get favorited cities - i think it should be renamed. in general, no route should be just "/"